src/app/item-popup/ItemStat.tsx

```diff
@@ -131,41 +164,52 @@ export function D1QualitySummaryStat({ item }: { item: D1Item }) {
   );
 }
 
+function getPlugHashesFromCategory(category: DimSocketCategory) {
+  return category.sockets
+    .map((socket) => {
+      return idx(socket, (socket) => socket.plug.plugItem.hash) || null;
+    })
+    .filter((hash) => !!hash);
+}
+
 /*
- * Looks through the item sockets to find any weapon/armor mods that modify this stat.
- * Returns the total value the stat is modified by, or 0 if it is not being modified.
+ * Gets all sockets that have a plug which doesn't get grouped in the Reusable socket category.
+ * The reusable socket category is used in armor 1.0 for perks and stats.
  */
-function getModdedStatValue(item: DimItem, stat: DimStat) {
+function getNonReuseableModSockets(item: DimItem) {
   if (!item.isDestiny2() || !item.sockets) {
-    return 0;
+    return [];
   }
 
   const reusableSocketCategory = item.sockets.categories.find((category) => {
     return category.category.categoryStyle === DestinySocketCategoryStyle.Reusable;
   });
 
   const reusableSocketHashes =
-    (reusableSocketCategory &&
-      reusableSocketCategory.sockets.map((socket) => {
-        return idx(socket, (socket) => socket.plug.plugItem.hash) || null;
-      })) ||
-    [];
-
-  const modSockets =
-    item.sockets.sockets.filter((socket) => {
-      const plugItemHash = idx(socket, (socket) => socket.plug.plugItem.hash) || null;
-      const categoryHashes = idx(socket, (socket) => socket.plug.plugItem.itemCategoryHashes) || [];
-      return (
-        _.intersection(categoryHashes, modItemCategoryHashes).length > 0 &&
-        // exclude the socket if it is "reusable" ie. selectable armor stats pre 2.0
-        !reusableSocketHashes.includes(plugItemHash) &&
-        // we only care about the ones that modify this stat
-        Object.keys(idx(socket, (socket) => socket.plug.stats) || {}).includes(
-          String(stat.statHash)
-        )
-      );
-    }) || [];
+    (reusableSocketCategory && getPlugHashesFromCategory(reusableSocketCategory)) || [];
+
+  return item.sockets.sockets.filter((socket) => {
+    const plugItemHash = idx(socket, (socket) => socket.plug.plugItem.hash) || null;
+    const categoryHashes = idx(socket, (socket) => socket.plug.plugItem.itemCategoryHashes) || [];
+    return (
+      _.intersection(categoryHashes, modItemCategoryHashes).length > 0 &&
+      !reusableSocketHashes.includes(plugItemHash)
+    );
+  });
+}
+
+/*
```

Comment:
If you make comments with `/** */` (note the two stars at the start) then editors like VSCode will show that documentation in autocomplete.